azure-mgmt-adhybridhealthservice/azure/mgmt/adhybridhealthservice/operations/ad_domain_service_members_operations.py

```diff
@@ -0,0 +1,129 @@
+    :ivar api_version: The version of the API to be used with the client request. Constant value: "2014-01-01".
+    """
+
+    models = models
+
+    def __init__(self, client, config, serializer, deserializer):
+
+        self._client = client
+        self._serialize = serializer
+        self._deserialize = deserializer
+        self.next_partition_key = ""
+        self.next_row_key = ""
+        self.api_version = "2014-01-01"
+
+        self.config = config
+
+    def list(
+            self, service_name, is_groupby_site, filter=None, query=None, take_count=None, custom_headers=None, raw=False, **operation_config):
+        """Gets the details of the servers, for a given Active Directory Domain
+        Service, that are onboarded to Azure Active Directory Connect Health.
+
+        :param service_name: The name of the service.
+        :type service_name: str
+        :param is_groupby_site: Indicates if the result should be grouped by
+         site or not.
+        :type is_groupby_site: bool
+        :param filter: The server property filter to apply.
+        :type filter: str
+        :param query: The custom query.
+        :type query: str
+        :param take_count: The take count , which specifies the number of
+         elements that can be returned from a sequence.
+        :type take_count: int
+        :param dict custom_headers: headers that will be added to the request
+        :param bool raw: returns the direct response alongside the
+         deserialized response
+        :param operation_config: :ref:`Operation configuration
+         overrides<msrest:optionsforoperations>`.
+        :return: An iterator like instance of AddsServiceMember
+        :rtype:
+         ~azure.mgmt.adhybridhealthservice.models.AddsServiceMemberPaged[~azure.mgmt.adhybridhealthservice.models.AddsServiceMember]
+        :raises: :class:`CloudError<msrestazure.azure_exceptions.CloudError>`
+        """
+        def internal_paging(next_link=None, raw=False):
+
+            if not next_link:
+                # Construct URL
+                url = self.list.metadata['url']
+                path_format_arguments = {
+                    'serviceName': self._serialize.url("service_name", service_name, 'str')
+                }
+                url = self._client.format_url(url, **path_format_arguments)
+
+                # Construct parameters
+                query_parameters = {}
+                if filter is not None:
+                    query_parameters['$filter'] = self._serialize.query("filter", filter, 'str')
+                query_parameters['isGroupbySite'] = self._serialize.query("is_groupby_site", is_groupby_site, 'bool')
+                if query is not None:
+                    query_parameters['query'] = self._serialize.query("query", query, 'str')
+                query_parameters['nextPartitionKey'] = self._serialize.query("self.next_partition_key", self.next_partition_key, 'str')
```

Comment:
@lmazuel - Do you know if this query parameter formatting works when the value is an empty string?